extensions/amp-connatix-player/OWNERS.yaml

```diff
@@ -0,0 +1,5 @@
+# For an explanation of the OWNERS.yaml rules and syntax, see:
+# https://github.com/ampproject/amp-github-apps/blob/master/owners/OWNERS.example.yaml
+
+- mariusszabo
```

Comment:
This is a new owner, correct?

Reply:
These are the original PR authors, not sure if they should be included

Reply:
Original authors of what PR? Did they create this directory or did they just author a PR adding themselves as an owner?

Reply:
They created the directory but did not include an owners file originally

Reply:
SGTM